src/main.rs

```diff
@@ -471,27 +475,28 @@ fn backtrace(
             .unwrap_or("<unknown>");
         println!("{:>4}: {:#010x} - {}", frame, pc, name);
 
-        let fde = debug_frame.fde_for_address(bases, pc.into(), DebugFrame::cie_from_offset).with_context(|| {
-            "debug information is missing. Likely fixes:
-1. compile the Rust code with `debug = 1` or higher. This is configured in the `profile.*` section of Cargo.toml
-2. use a recent version of the `cortex-m` crates (e.g. cortex-m 0.6.3 or newer). Check versions in Cargo.lock
-3. if linking to C code, compile the C code with the `-g` flag"
-        })?;
-        let uwt_row = fde.unwind_info_for_address(&debug_frame, bases, ctx, pc.into())?;
-
-        let cfa_changed = registers.update_cfa(uwt_row.cfa())?;
+        let mut cfa_changed = false;
+        if let Ok(uwt_row) =
+            debug_frame.unwind_info_for_address(bases, ctx, pc.into(), DebugFrame::cie_from_offset)
+        {
+            cfa_changed = registers.update_cfa(uwt_row.cfa())?;
 
-        for (reg, rule) in uwt_row.registers() {
-            registers.update(reg, rule)?;
+            for (reg, rule) in uwt_row.registers() {
+                registers.update(reg, rule)?;
+            }
         }
 
+        // We might not find unwind info for some PC values. In that case, we assume that unwinding
```

Comment:
"PC values with no unwind info" could be external assembly with no CFI instructions or external C not compiled with the `-g` flag (I think). Those may not have trivial unwinding.